core/client/fs/src/main/java/alluxio/client/file/RetryHandlingFileSystemMasterClient.java

```diff
@@ -114,7 +114,7 @@ protected void afterConnect() {
       final CheckConsistencyPOptions options) throws AlluxioStatusException {
     return retryRPC(() -> {
       List<String> inconsistentPaths = mClient.checkConsistency(CheckConsistencyPRequest
-          .newBuilder().setPath(path.getPath()).setOptions(options).build())
+          .newBuilder().setPath(path.toString()).setOptions(options).build())
```

Comment:
What are the new requirements for paths sent over the wire, is there any time we should use `getPath` now?

Reply:
It looks like `getPath()` is more efficient than `toString()`. So I wrote a helper to call `getPath()` when the URI has no scheme, which we assume to be Alluxio path. And call `toString()` when it's transparent URI.